RecoEgamma/EgammaElectronProducers/plugins/ElectronSeedProducer.cc

```diff
@@ -328,6 +328,17 @@ ElectronSeedProducer::fillDescriptions(edm::ConfigurationDescriptions& descripti
     psd0.add<double>("hOverEHFMinE",0.8);
     psd0.add<double>("DeltaPhi2B",0.008);
     psd0.add<double>("PhiMin2B",-0.002);
+
+    psd3.add<std::string>("ComponentName",std::string("SeedFromConsecutiveHitsCreator"));
+    psd3.add<std::string>("propagator",std::string("PropagatorWithMaterial"));
+    psd3.add<double>("SeedMomentumForBOFF",5.0);
+    psd3.add<double>("OriginTransverseErrorMultiplier",1.0);
+    psd3.add<double>("MinOneOverPtError",1.0);
+    psd3.add<std::string>("magneticField",std::string(""));
+    psd3.add<std::string>("TTRHBuilder",std::string("WithTrackAngle"));
+    psd3.add<bool>("forceKinematicWithRegionDirection",false);
+    psd0.add<edm::ParameterSetDescription>("SeedCreatorPSet",psd3);
```

Comment:
ciao martin

      SeedCreatorPSet = cms.PSet(
            ComponentName                     =
cms.string('SeedFromConsecutiveHitsCreator'),
            MinOneOverPtError                 = cms.double(1.0),
            OriginTransverseErrorMultiplier   = cms.double(1.0),
            SeedMomentumForBOFF               = cms.double(5.0),
*            TTRHBuilder                       =
cms.string('hltESPTTRHBWithTrackAngle'),*
            forceKinematicWithRegionDirection = cms.bool(False),
            magneticField                     = cms.string(''),
            propagator                        =
cms.string('PropagatorWithMaterial')
      ),

should be the config as the one currently used @HLT
thanks much
  mia

On Wed, Mar 18, 2015 at 10:36 PM, Martin Grunewald <notifications@github.com
> wrote:

> In RecoEgamma/EgammaElectronProducers/plugins/ElectronSeedProducer.cc
> <https://github.com/cms-sw/cmssw/pull/8367#discussion_r26710386>:
>
> > @@ -328,6 +328,17 @@ ElectronSeedProducer::fillDescriptions(edm::ConfigurationDescriptions& descripti
> >      psd0.add<double>("hOverEHFMinE",0.8);
> >      psd0.add<double>("DeltaPhi2B",0.008);
> >      psd0.add<double>("PhiMin2B",-0.002);
> > +
> > +    psd3.add<std::string>("ComponentName",std::string("SeedFromConsecutiveHitsCreator"));
> > +    psd3.add<std::string>("propagator",std::string("PropagatorWithMaterial"));
> > +    psd3.add<double>("SeedMomentumForBOFF",5.0);
> > +    psd3.add<double>("OriginTransverseErrorMultiplier",1.0);
> > +    psd3.add<double>("MinOneOverPtError",1.0);
> > +    psd3.add<std::string>("magneticField",std::string(""));
> > +    psd3.add<std::string>("TTRHBuilder",std::string("WithTrackAngle"));
> > +    psd3.add<bool>("forceKinematicWithRegionDirection",false);
> > +    psd0.add<edm::ParameterSetDescription>("SeedCreatorPSet",psd3);
>
> Mia, what values should be put for the HLT for the parameters of the new
> SeedCreatorPSet in the ElectronSeedProducer?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8367/files#r26710386>.
>